README.md

```diff
@@ -123,18 +125,21 @@ cd ~/
 OSX: mkdir ~/Dev; cd ~/Dev
 Windows: <Ignore This>
 
-# Start ssh-agent and load your key
```

Comment:
MUST: 

So this clearly is not necessary for the KDK, but unfortunately some people in our don't have the ssh skills to be able to access github and bitbucket via ssh within the KDK.  For this they need ssh-agent, and they need it either spelled out or automated. 

Can you figure out a way to either automate this or document it in an Appendix section?

Reply:
Perhaps we can build it into the kdk

Reply:
In a different PR.